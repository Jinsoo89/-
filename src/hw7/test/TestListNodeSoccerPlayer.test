# Test the parser on a graph that contains several nodes.

# load graph
LoadGraph players soccerPlayers.tsv

# list nodes in the graph
ListNodes players