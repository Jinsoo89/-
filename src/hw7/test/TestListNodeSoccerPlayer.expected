# Test the parser on a graph that contains several nodes.

# load graph
loaded graph players

# list nodes in the graph
players contains: C.Jinsoo C.Ronaldo D.Alves D.Costa D.Maria Griezmann Isco L.Messi L.Modric Neymar S.Ramos Z.Ibrahimovic